brayns/utils/image/codecs/JpegCodec.cpp

```diff
@@ -32,6 +32,7 @@ std::string JpegCodec::getFormat() const
 
 std::string JpegCodec::encode(const Image &image, int quality) const
 {
+    quality = std::max(100 - std::max(quality, 0), 0);
```

Comment:
We already check that at the JSON-RPC level, so if the quality is invalid, it would be an error from our side no ?

In this case an assert would avoid hiding a bug.

Reply:
The error here is that 0 is the lowest quality for JPG, while it is the maximum for PNG, so this fixes the parameter so it means the same for both formats.

Reply:
I don't think stbi uses quality for PNG but from the WebAPI we use 0 = worst and 100 = best no ?

So we should change that in the PngCodec, not the JPEG no ?

Reply:
When Brayns was using FreeImage, that parameter has been used as PNG compression rate. 0 meant no compression, which gave bigger files and faster decoding.

Reply:
The best solution IMO would be to rename the parameter to compression (as it is more accurate and clear). For some reason, however, stbi has the png compression ratio hardcoded as a global variable, but can be edited before every call to to the encoder.

In this specific case, the tests where failing. The reason is that JPG quality on stbi ranges from 1 to 100. When inputing 0 (default quality we use in Brayns), stbi uses by default 90, which reduces a bit the quality. This resulted on smaller files, but the ImageValidator failed to pass the test.